metron-deployment/roles/kibana/README.md

```diff
@@ -1,35 +0,0 @@
-Kibana 4
---------
-
-This role installs Kibana along with the default Metron Dashboard.
-
-### FAQ
-
-#### How do I change Metron's default dashboard?
```

Comment:
If you know where you want it, I'm actually writing that up as part of METRON-745 (which needed to use that module).